src/libraries/System.Text.Json/src/System/Text/Json/BitStack.cs

```diff
@@ -110,13 +110,13 @@ public bool Pop()
             }
             else
             {
-                inObject = PopFromArray();
+                inObject = PeekInArray();
             }
             return inObject;
         }
 
         [MethodImpl(MethodImplOptions.NoInlining)]
-        private bool PopFromArray()
+        private readonly bool PeekInArray()
```

Comment:
What are we peeking at in this case? It seems we're just returning a boolean so perhaps this could be expressed as a property?

Reply:
It looks at the top of the stack assuming the stack is backed by an array. `IsInArray` sounds like it's checking containment. I also like it being a method since it's symmetric with Push/PushToArray and Peek/PeekInArray. I'm open to better naming but I left it as is with more comments.